power/nut-client/pkg.yaml

```diff
@@ -0,0 +1,101 @@
+name: nut-client
+variant: scratch
+shell: /toolchain/bin/bash
+dependencies:
+  - stage: base
+  - image: "{{ .PKGS_PREFIX }}/openssl:{{ .PKGS_VERSION }}"
+  - image: "{{ .PKGS_PREFIX }}/util-linux:{{ .PKGS_VERSION }}"
+steps:
+  - sources:
+      - url: https://networkupstools.org/source/2.8/nut-2.8.0.tar.gz
+        destination: nut.tar.gz
+        sha256: c3e5a708da797b7c70b653d37b1206a000fcb503b85519fe4cdf6353f792bfe5
+        sha512: 3c413ae54088045a713eb80cf1bdda474f41bb3b67c7c0248aa7a0c4d441dce1ff42627a2735273d7e36892d1f2eeb895220cf28af63fec2fa0c7a267f82d577
+    prepare:
+      - |
+        sed -i 's#$VERSION#{{ .VERSION }}#' /pkg/manifest.yaml
+      - |
+        tar -xzf nut.tar.gz --strip-components=1
+      - |
+        # no shell for you
+        patch -p1 < /pkg/replace_system.patch
```

Comment:
patches are usually kept in a `patches` folder

Reply:
updated